Library/Formula/synergy.rb

```diff
@@ -0,0 +1,63 @@
+class Synergy < Formula
+  homepage "http://synergy-project.org"
+  url "https://github.com/synergy/synergy/archive/1.6.3.tar.gz"
+  sha256 "93b1965b8e0cfc55cc654aab7053a58b9e730d09e68bbc520be32353a01dce1a"
+
+  depends_on "cmake" => :build
+  depends_on "qt5" => :build
+
+  patch :DATA
```

Comment:
We prefer to wait for patches to get merged before adding them to formulae, particularly new formulae. I don't personally consider it ready and working as long as it requires patches that upstream hasn't accepted. Once the patches are accept we can consider using the upstream commits as the patches.